assets/css/sensei-course-theme/contact-teacher.scss

```diff
@@ -33,23 +33,22 @@
 	}
 }
 
-.sensei-contact-teacher-success {
-	font-family: 'Inter', sans-serif;
-	display: block;
-	position: absolute;
-	top: 0;
-	right: 0;
-	bottom: 0;
-	left: 0;
-	padding-top: 30px;
-	background-color: #fff;
```

Comment:
We need this. Could be set to `var(--bg-color)`.

https://user-images.githubusercontent.com/2578542/150340582-74995abe-18c9-455a-b720-17535ca6ef0e.mp4

Reply:
That's a whoopsie, fixing in e16e420